cmd/limactl/copy.go

```diff
@@ -128,14 +154,28 @@ func copyAction(cmd *cobra.Command, args []string) error {
 			return err
 		}
 	}
+
 	sshArgs := sshutil.SSHArgsFromOpts(sshOpts)
 
-	sshCmd := exec.Command(arg0, append(sshArgs, scpArgs...)...)
+	sshCmd := exec.Command(arg0, createArgs(sshArgs, copyToolArgs, defaultTool)...)
 	sshCmd.Stdin = cmd.InOrStdin()
 	sshCmd.Stdout = cmd.OutOrStdout()
 	sshCmd.Stderr = cmd.ErrOrStderr()
-	logrus.Debugf("executing scp (may take a long time): %+v", sshCmd.Args)
+	logrus.Debugf("executing %s (may take a long time): %+v", arg0, sshCmd.Args)
 
 	// TODO: use syscall.Exec directly (results in losing tty?)
 	return sshCmd.Run()
 }
+
+func isCopyToolRsync(copyTool copyTool) bool {
+	return copyTool == Rsync
+}
```

Comment:
This helper is not helpful for 2 reasons:
- We should have exactly one check for the tool
- Even if we need checks it does not help